src/exponential.jl

```diff
@@ -97,7 +97,40 @@ function expm_overapproximation(A::IntervalMatrix{T, Interval{T}}, t, p) where {
     return res
 end
 
+# Implementation of Prop. 1 in Althoff, Matthias, Bruce H. Krogh, and Olaf Stursberg.
+# "Analyzing reachability of linear dynamic systems with parametric uncertainties."
+# Modeling, Design, and Simulation of Systems with Uncertainties. Springer, Berlin, Heidelberg, 2011. 69-94.
 function _expm_remainder(A::IntervalMatrix{T}, t, p; n=checksquare(A)) where {T}
+    C = max.(abs.(inf.(A)), abs.(sup.(A)))
+    M = exp(C*t)
+
+    # compute Q = I + Mt + M^2/2! + ... + M^p/p!
+    # where M = exp(abs(A*t)) and abs is taken component-wise
```

Comment:
The paper uses `C` and not `M` in the sum.

Reply:
Yes, sure, it was a silly mistake. Thanks for finding it.